bucket/gsmartcontrol.json

```
@@ -0,0 +1,48 @@
+{
+    "version": "1.1.3",
+    "description": "Hard disk drive and SSD health inspection tool",
+    "homepage": "https://gsmartcontrol.sourceforge.io/home/",
+    "license": "GPL-3.0-only",
+    "suggest": {
+        "Smartmontools": "extras/smartmontools"
```

It is `main/smartmontools` not `extras/smartmontools`
```suggestion
        "Smartmontools": "main/smartmontools"
```

---

Also there's https://github.com/ScoopInstaller/Extras/issues/7532 which is not yet fixed.

---

Also there's https://github.com/ScoopInstaller/Extras/issues/7532 which is not yet fixed.

---

> It is `main/smartmontools` not `extras/smartmontools`

Thanks!